singleton-service/src/service/auth.rs

```diff
@@ -0,0 +1,86 @@
+use log::debug;
+use threescale::structs::AppIdentifier;
+use threescalers::{
+    api_call::{ApiCall, Kind},
+    application::Application,
+    credentials::*,
+    extensions::{self},
+    http::Request,
+    service::Service,
+    transaction::Transaction,
+};
+
+/// Proxy level representation for authorization request.
+pub struct Auth {
+    pub service_id: String,
+    pub service_token: String,
+    pub app_id: AppIdentifier,
+}
```

Comment:
You are publishing and providing getters as well. Have either not both.

Reply:
fixed